src/libraries/Microsoft.Extensions.Logging.EventLog/src/WindowsEventLog.cs

```diff
@@ -4,9 +4,12 @@
 using System;
 using System.Diagnostics;
 using System.Security;
+using System.Runtime.Versioning;
 
 namespace Microsoft.Extensions.Logging.EventLog
 {
+#pragma warning disable CA1416 // Debug.Assert(OperatingSystem.IsWindows()) is not available
```

Comment:
Please reference an issue number if we have one

Reply:
Same as the previous, this assembly only targets netstandard so the relevant APIs aren't available.